frappe/__init__.py

```diff
@@ -1850,6 +1850,14 @@ def get_list(doctype, *args, **kwargs):
 	        # filter as a list of dicts
 	        frappe.get_list("ToDo", fields="*", filters = {"description": ("like", "test%")})
 	"""
+	
+	try:
+		meta = get_meta(doctype)
+	except exceptions.DoesNotExistError:
+		# We only need this for the installing case, but it is necessary. Please keep it
+		meta = False
+	if meta and meta.get("is_virtual"):
```

Comment:
This isn't the right way to do things... why meta can raise `DoesNotExistError` here? 


Meta should definitely exist for virtual doctype too, this change isn't addressing root cause but using error as business logic to handle it somehow 😅 


<img width="446" alt="Screenshot 2022-07-19 at 3 50 26 PM" src="https://user-images.githubusercontent.com/9079960/179728354-ecc738f8-062c-41e4-9410-04ab3e8992c6.png">

Reply:
Can you please first expand PR description to explain what is the problem here?